corehq/messaging/scheduling/tests/test_content.py

```diff
@@ -212,6 +215,28 @@ def test_sms_language_fallback(self):
             message_dict['*']
         )
 
+    def test_email_validation_valid(self):
+        email = RecipientWithEmail("test@example.com")
+        EmailContent().get_recipient_email(email)
+
+    def test_email_validation_empty_email(self):
+        email = RecipientWithEmail("")
+        with self.assertRaises(ContentException) as e:
+            EmailContent().get_recipient_email(email)
+        self.assertEqual(e.exception.error_type, MessagingEvent.ERROR_NO_EMAIL_ADDRESS)
+
+    def test_email_validation_no_email(self):
+        email = RecipientWithEmail(None)
+        with self.assertRaises(ContentException) as e:
+            EmailContent().get_recipient_email(email)
+        self.assertEqual(e.exception.error_type, MessagingEvent.ERROR_NO_EMAIL_ADDRESS)
+
+    def test_email_validation_invalid_email(self):
+        email = RecipientWithEmail("bob")
+        with self.assertRaises(ContentException) as e:
+            EmailContent().get_recipient_email(email)
+        self.assertEqual(e.exception.error_type, MessagingEvent.ERROR_INVALID_EMAIL_ADDRESS)
```

Comment:
do we need a test for `additional_error_text`?

Reply:
I don't think it's necessary